R/RunningModels.R

```diff
@@ -0,0 +1,840 @@
+globalVariables(names=c("XPep", "XProt", "Y"))
```

Comment:
I'm quite ignorant regarding global variables but you may want to be careful in case these would exist in the user's environment once the package is loaded (I think that's the case for global variables?). Do you really need them? Would local variables be enough for your purposes?